examples/counter/sources/counter.move

```diff
@@ -6,8 +6,12 @@ module rooch_examples::counter {
       value:u64,
    }
 
-   fun init(ctx: &mut StorageContext, account: &signer){
-      account_storage::global_move_to(ctx, account, Counter{value:0});
+   public fun init_for_test(ctx: &mut StorageContext, account: &signer) {
```

Comment:
Is it possible to make the `init_for_test` `#[test-only]` in the future?

Reply:
Yes, we can add `#[test_only]` to prevent a function from being compiled.

By passing a special flag during the construction of the compiler in the Parse phase, this can be resolved. 

I will add this feature in the next PR.